src/components/indicators/z-stepper-item/styles.css

```diff
@@ -0,0 +1,114 @@
+:host {
+  position: relative;
+  font-family: var(--dashboard-font);
+  font-weight: var(--font-rg);
+}
+
+:host(:not(:last-child)) {
+  flex: 1;
+}
+
+:host(:not(:last-child))::after {
+  position: absolute;
+  top: var(--basex5);
+  left: calc(var(--basex4) + var(--basex4));
+  right: var(--basex2);
+  z-index: 1;
+  content: '';
+  display: block;
+  width: auto;
+  margin: 0 calc(-1 * var(--basex2)) 0 var(--basex2);
+  border-bottom: solid 2px var(--bg-grey-700);
+}
+
+.stepper-item {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  width: calc(var(--basex6) + var(--basex4));
+  padding: 0;
+  color: inherit;
+  white-space: nowrap;
+  font-family: inherit;
+  font-size: 18px;
+  text-decoration: none;
+  line-height: 28px;
+  background: transparent;
+  border: 0;
+  outline: 0;
+  appearance: none;
+}
+
+:host(:not([disabled])) .stepper-item {
+  cursor: pointer;
+}
+
+:host([disabled]) .stepper-item {
+  pointer-events: none;
+}
+
+.indicator {
+  content: counter(stepper);
+  box-sizing: border-box;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  width: var(--basex6);
+  height: var(--basex6);
+  margin: var(--basex2);
+  font-size: 16px;
+  font-weight: var(--font-sb);
+  line-height: 1;
+  color: var(--myz-blue);
+  background: var(--bg-white);
+  border: 2px solid var(--myz-blue);
+  border-radius: 50%;
+}
+
+:host(:not([disabled]):not([pressed]):hover) .indicator {
+  color: var(--myz-blue-dark);
+  border-color: var(--myz-blue-dark);
+}
+
+:host(:not([disabled]):not([pressed]):focus-within) .indicator {
+  color: var(--myz-blue);
+  border-color: var(--myz-blue);
+  box-shadow: 0px 0px 2px 2px var(--accent-dark);
+}
+
+:host([pressed]:not([disabled])) .indicator {
+  color: var(--text-white);
+  background: var(--myz-blue);
+  border-color: var(--myz-blue);
+  box-shadow: none;
+}
+
+:host([disabled]) .indicator {
+  color: var(--text-grey-800);
+  background: var(--bg-grey-200);
+  border-color: var(--text-grey-800);
+}
+
+/* Mobile breakpoint */
+@media only screen and (max-width: 768px) {
```

Comment:
CSS should be mobile first (if you create components with `yarn generate` the file is created with breakpoints for instance `@media only screen and (min-width: 768px) {}`). 
Also, 768px should still be tablet.

Reply:
The comment was wrong but it seems this is an old version of the media query 🤔

Reply:
I'm not sure :see_no_evil:

Reply:
The media query should had changed in this commit: https://github.com/ZanichelliEditore/design-system/commit/1c89f7cdbee714ea1fb72ba63c685671296a538e#diff-3922f519e39d8c8f05e9d120afa4caf0R99

Anyway, I think it's ok now :)

Reply:
Haha the commit is old but I think you pushed it this morning. Great! :smile: